overlord/snapstate/autorefresh_test.go

```diff
@@ -206,6 +208,53 @@ func (s *autoRefreshTestSuite) TestRefreshManagedTimerWins(c *C) {
 	c.Check(err, IsNil)
 }
 
+func (s *autoRefreshTestSuite) TestRefreshManagedDenied(c *C) {
```

Comment:
it would also be good to see that we still get a message from the transition, i.e. we have `refresh.schedule=<date>`, then we change it to `refresh.schedule=managed` (with CanManageRefreshes returning true this time), and then we see the `refresh is managed via the snapd-control interface` message, then making CanManageRefreshes return false, and see that the `managed refresh schedule denied, no properly configured snapd-control` message shows up. 

Perhaps this test is already here and I didn't see it when looking ?

Reply:
I've added a test that went through a transition, but it mostly regurgitated what the other tests are already doing. Instead I've tweaked the test that checks a scenario when can-manage is true to also verify that a relevant message is logged.